docs/examples/site_config.yml

```diff
@@ -0,0 +1,225 @@
+          # the |- strips off the trailing newline in the YAML block
+          default: |-
+            {%- if VARS.DOCKER_IMAGE is defined -%}
+            {{ VARS.DOCKER_IMAGE }}
+            {%- else -%}
+            1234567890.dkr.ecr.us-east-1.amazonaws.com/{{ APP.group }}{{ ( ':' ~ VARS.IMAGE_TAG ) if VARS.IMAGE_TAG is defined else '' }}
+            {%- endif -%}
+        # Set reasonable defaults for 'cpus', 'mem', and 'disk'
+        cpus:
+          default: 0.5
+        mem:
+          default: 512
+        disk:
+          default: 10
+        instances:
+          default: 3
+        # Default constraints make sure service instances are spread out across availability zones
+        constraints:
+          default:
+            - [ "@region", "IS", "aws/us-east-1" ]
+            - [ "@zone", "GROUP_BY", "3" ]
+        ports:
+          fields:
+            # Add custom 'vip_port' field for use below
+            vip_port:
+              type: int
+            # Automatic port labels when a VIP port is defined
+            labels:
+              default_action: prepend
+              default:
+                - name: VIP_{{ port_index }}
+                  value: /{{ APP.id }}:{{ port.vip_port }}
+                  condition: port.vip_port
+        port_definitions:
+          fields:
+            # Add custom 'vip_port' field for use below
+            vip_port:
+              type: int
+            # Automatic port labels when a VIP port is defined
+            labels:
+              default_action: prepend
+              default:
+                - name: VIP_{{ port_index }}
+                  value: /{{ APP.id }}:{{ port.vip_port }}
+                  condition: port.vip_port
+        # Automatically add labels for service group/name
+        container_labels:
+          default_action: prepend
+          default:
+            - service={{ APP.group }}
+            - service_component={{ APP.name }}
+        # Provide fetch config for ECR auth
+        fetch:
+          locked: true
+          default:
+            - uri: "{{ VARS.DOCKER_CREDENTIALS_URI }}"
+              extract: true
+              executable: false
+              cache: false
+              condition: VARS.DOCKER_CREDENTIALS_URI is defined and not APP.no_fetch
+        # Provide method for disabling default fetch config
+        # We need an explicit flag, because the deploy tool can't tell the difference
+        # between an explicit empty list provided and no value provided at all
+        no_fetch:
+          type: bool
+          default: false
+        upgrade_strategy:
+          fields:
+            minimum_health_capacity:
+              default: 1
+            maximum_over_capacity:
+              default: 1
+        secrets:
+          fields:
+            source:
+              default: '{{ secret.name }}'
+              transform:
+                prefix: '{{ VARS.SERVICE_NAME }}/'
```

Comment:
`secrets` here?

Reply:
This is the generic Marathon/Metronome `secrets` stuff for DC/OS enterprise, so this is fine to stay.